recipe/meta.yaml

```diff
@@ -51,7 +51,7 @@ requirements:
     - libtiff
     - libcurl
     - libpng
-    - ffmpeg
+    - ffmpeg 4.4
```

Comment:
Can you please leave a comment here about the situation and/or pointing to #29

Reply:
Done